src/coprocessor_v2/plugin_registry.rs

```diff
@@ -153,9 +159,8 @@ impl PluginRegistry {
                             rename(&old_file, &new_file);
                         }
                         Ok(DebouncedEvent::Write(file)) => {
-                            warn!("another process is overwriting a coprocessor plugin while the plugin is loaded. This can lead to severe issues!"; "plugin_path" => ?file);
+                            warn!("another process is overwriting a coprocessor plugin while the plugin is loaded. Unloading the plugin..."; "plugin_path" => ?file);
                             unload(&file);
```

Comment:
After this change, do we really need to unload the plugin?

> To solve this problem, this PR simply unloads the plugin whenever it removed or changed, and forbid future reloading from the same path.

Unloading a plugin may cause problems. Imagine that there is a tokio future running in the runtime, and the plugin disappears. Then the program would crash. Since a plugin cannot be released for now, what about also removing `unload` here?

Reply:
I concerned following behaviours:

||file renamed| file changed | file deleted |
|--|----|---|---|
|behaviour 1|unload|unload|unload|
|behaviour 2|do nothing|unload|unload|
|behaviour 3|do nothing|do nothing|unload|
|behaviour 4|do nothing|do nothing|do nothing|

In this PR we have behaviour 2. I think it will be useful if users can unload a plugin, and "file changed" and "file deleted" both implies the original file is not needed.

Do you mean we should use behaviour 3? If `unload` is a dangerous operation, should we use behaviour 4? I'm not sure which is more sensible.

Reply:
I would prefer behavior 4 indeed 🤣🤣 what do you think @andylokandy

Reply:
Given that we can neither lock the plugin file nor reload when the plugin file is recreated, we can not do much. I prefer behavior 4 because it's simpler for the user, but we can print a warning instead of doing nothing.

Reply:
Ok. Then I will remove all unload related code.